src/server/editorServices.ts

```diff
@@ -1006,14 +1006,15 @@ namespace ts.server {
 
         /**
          * @param filename is absolute pathname
+         * @param fileContent is a known version of the file content that is more up to date

```

Comment:
More up to date than what? The content stored on the host, right? Can you update the comment with that detail?